cardano-node/test/Test/Common/Base.hs

```diff
@@ -15,6 +15,8 @@ module Test.Common.Base
 
 import           Control.Monad
 import           Control.Monad.IO.Class (liftIO)
+import           Control.Monad.Morph
+import           Control.Monad.Trans.Resource
```

Comment:
Everything that is imported from outside the current repo should used either qualified or explicit imports.

Reply:
Chose to use explicit imports.